lemminx-liberty/src/test/java/io/openliberty/LibertyDiagnosticTest.java

```diff
@@ -687,4 +687,51 @@ public void testConfigElementVersionLess() throws JAXBException {
         configForMissingFeature.setMessage(LibertyDiagnosticParticipant.MISSING_CONFIGURED_FEATURE_MESSAGE);
         XMLAssert.testDiagnosticsFor(serverXML1, null, null, serverXMLURI,configForMissingFeature);
     }
+
+    @Test
+    public void testUniquenessForNameChangedFeatureDiagnostic() throws BadLocationException {
+        String serverXML = String.join(newLine, //
+                "<server description=\"Sample Liberty server\">", //
+                "       <featureManager>", //
+                "               <feature>jms-2.0</feature>", //
+                "               <feature>messaging-3.0</feature>", //
+                "       </featureManager>", //
+                "</server>" //
+        );
+        Diagnostic invalid = new Diagnostic();
+        invalid.setRange(r(3, 24, 3, 37));
+        invalid.setMessage("ERROR: The messaging-3.0 feature cannot be configured with the jms-2.0 feature because they are two different versions of the same feature. The feature name changed from jms to messaging for Jakarta EE. Remove one of the features.");
```

Comment:
Can you use a feature with mixed case to prove that the code issuing this diagnostic is preserving the case of the feature names?

Reply:
Already added a test for this with ejb and enterpriseBeans . But this was versionless. I will add one more with specifying version for ejb and enterpriseBeans